src/generated/apps_generated.rs

```diff
@@ -49,6 +50,7 @@ pub enum AppCredential {
     Secret(String),
     Certificate(Blob),
     TrustedCa(TrustAnchor),
+    GoogleServiceAccount {},
```

Comment:
Why is this an empty struct?

Reply:
To be able to expand it in the future. You can also check the source that this code was generated from.